.github/PULL_REQUEST_TEMPLATE/contributor_checklist.md

```diff
@@ -0,0 +1,8 @@
+### Description
+<!-- FILL IN -->
+
+### Checklist
+- [ ] Wrote description of my changes above in the `CHANGELOG.md` file
```

Comment:
Wrote description of my changes above **and** in the `CHANGELOG.md` file

Reply:
Sure, feel free to do that and then merge this.

Reply:
Done